### ISSUE TITLE
Double superscripts in Tensor.display() output

### ISSUE THREAD

**Issue by chenpeizhi** [tschijnmo/drudge#9]
When a factor of the amplitude of a term is raised to a power, the LaTeX output given by Tensor.display() contains double superscripts, which cannot be properly displayed in a Jupyter notebook. It seems that this could be fixed by modifying the ScalarLatexPrinter class in report.py.

**Comment by tschijnmo** [comment]
Yes, I can remember this issue.  During a research on tensor structured CC theory, there was something like a regular expression to fix this in a really dirty way.  I'll see if there is some easy fix.
